enchanted-swiftie/cogs/songs.py

```diff
@@ -0,0 +1,66 @@
+import billboard
+import datetime as dt
+from discord.ext import commands
+
+
+class Songs(commands.Cog):
+    def __init__(self, bot):
+        self.bot = bot
+        self.singles_dict = {}
```

Comment:
the type checker will understand this is a dict, but we also want to specify the types of the keys and values.
```suggestion
        from typing import Dict

        self.singles_dict: Dict[str, int] = {}
```